### ISSUE TITLE
sample: socket client use

### ISSUE THREAD

**Issue by pmp-p** [pmp-p/pygbag#6]
minimal example would be connecting to a webirc lobby

**Comment by gabriel-v** [comment]
Any update on this one? Saw a [patch for `sock_connect`](https://github.com/pygame-web/python-wasm-sdk/commit/c6801c47f029837670a111c80fc3f88e03b01ba7)  so I bet we need to use [this](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.sock_connect) - but got some strange errors.

**Comment by pmp-p** [comment]
yeah i patched asyncio for socket use, but only to work with IoT / telemetrix-web
for irc no need to use asyncio, polling socket is ok since it's only done once per frame but sample is very crude and i won't have time for that until pycon fr is finished since i only present the IoT stuff.
anyway the [minimal irc sample is here](https://github.com/pygame-web/showroom/blob/main/pygame-scripts/aio.sockets.html)
